src/match_throttle_metadata_keys.c

```diff
@@ -242,30 +242,52 @@ static int mtg_create(const oconfig_item_t *ci, void **user_data)
     } else {
       int k;
       mtg_key_tracker_t *kt = ctx->key_tracker;  // Alias
-      static const char *int_keys[4] = {
+      static const char *size_t_keys[4] = {
```

Comment:
Should this be `2` instead of `4` now, or is it intentional? Similar with `char *cdtime_keys[4]` below.

Reply:
Nice catch. The explicit sizes were not needed in the first place, so I removed them completely.